snips_nlu/pipeline/configs/intent_classifier.py

```diff
@@ -72,6 +74,12 @@ def unit_name(cls):  # pylint:disable=no-self-argument
         from snips_nlu.intent_classifier import LogRegIntentClassifier
         return LogRegIntentClassifier.unit_name
 
+    def get_required_resources(self):
```

Comment:
Shouldn't we make an abstract class `ProcessingUnitWithResources` which would require this method to be implemented ??

Reply:
This is exactly what is done with `ProcessingUnitConfig`